sunpy/instr/tests/test_lyra.py

```diff
@@ -97,7 +98,9 @@ def lyra_ts():
 
 @pytest.mark.remote_data
 def test_remove_lytaf_events_from_timeseries(lyra_ts):
-    """Test if artefacts are correctly removed from a TimeSeries."""
+    """
+    Test if artefacts are correctly removed from a TimeSeries.
```

Comment:
"artefact" -> "artifact"? The latter seems to be the preferred American spelling, but I'm not sure what we've standardized on in the codebase as a whole...